spec/index_spec.rb

```diff
@@ -372,4 +372,51 @@
       )
     end
   end
+
+  describe Daru::CategoricalIndex do
+    before do
+      @idx = Daru::CategoricalIndex.new(
```

Comment:
@zverok I see that I can use "let" to replace before..do block but when and where to use "subject"? Its not clear to me. Please let me know.